datastream-server/src/main/java/com/linkedin/datastream/server/assignment/BroadcastStrategy.java

```diff
@@ -93,6 +107,13 @@ public BroadcastStrategy(Optional<Integer> maxTasks) {
     return newAssignment;
   }
 
+  private DatastreamTask getOrCreateDatastreamTask(List<DatastreamTask> reuseTasksPerDg, DatastreamGroup dg) {
+    if (reuseTasksPerDg.size() > 0) {
```

Comment:
Can be more succinctly expressed as `if (!reuseTasksPerDg.isEmpty())`

Reply:
Done.